docs/content/en/docs/workshops/packages/prometheus/prometheus_grafana.md

```diff
@@ -0,0 +1,200 @@
+---
+title: "Prometheus with Grafana"
+linkTitle: "Prometheus with Grafana"
+weight: 2
+date: 2022-12-09
+description: >  
+---
+
+This tutorial demonstrates how to config the Prometheus package to scrape metrics from an EKS Anywhere cluster, and visualize them in Grafana.
+
+This tutorial walks through the following procedures:
+- [Install the Prometheus package](#install-the-prometheus-package);
+- [Install Grafana helm charts](#install-grafana-helm-charts);
+- [Set up Grafana dashboards](#set-up-grafana-dashboards).
+
+## Install the Prometheus package
+
+The Prometheus package creates two components by default:
+
+- [Prometheus-server,](https://github.com/prometheus/prometheus) which collects metrics from configured targets, and stores the metrics as time series data;
+- [Node-exporter,](https://github.com/prometheus/node_exporter) which exposes a wide variety of hardware- and kernel-related metrics for prometheus-server (or an equivalent metrics collector, i.e. ADOT collector) to scrape.
+
+The `prometheus-server` is pre-configured to scrape the following targets at `1m` interval:
+- Kuberenetes API servers
+- Kuberenetes nodes
+- Kuberenetes nodes cadvisor
+- Kuberenetes service endpoints
+- kubernetes services
+- kubernetes pods
+- prometheus-server itself
```

Comment:
You do capitalize Prometheus-server earlier. I think it would be good to do so again here since it seems more of a title for the feature than the actual component name.

Reply:
Sounds good. Updated